aptos-move/move-examples/fungible_token/sources/managed_fungible_token.move

```diff
@@ -0,0 +1,175 @@
+module fungible_token::managed_fungible_token {
```

Comment:
Can you add high-level comments explaining what a fungible token is and how it's created? Also descriptions of the different functionalities provided here would be great